src/DefinitionResolver.php

```diff
@@ -234,6 +235,18 @@ public function createDefinitionFromNode(Node $node, string $fqn = null): Defini
             $def->documentation = $this->getDocumentationFromNode($node);
         }
 
+        $def->parameters = [];
+        if (property_exists($node, 'parameters') && $node->parameters) {
```

Comment:
@roblourens could you add an interface for `FunctionHeader` or callables so this can be checked for with `instanceof`?